### ISSUE TITLE
[Enhancement] Math calculations of day

### ISSUE THREAD

**Issue by se7enek** [SkyFire55/daily-rewards#18]
Simple enough, imho easy to add.

Instead of making config for "Day: "1"", make it somewhere close to it (example below)

Day = 7 (day 7, simple enough)
Day % 7 = 0 (whenever remainder of dividing current day by 7 is equal to 0, so.. 14,21,28... (skipping day 7 cuz declared above)
Day > 5 (day 6,_,8,9,10,11,12,13,_,15 etc..., skipping both day 7 and every multiplification of 7)

That way player could do special functions with never-ending daily rewards.

Possible to do ? @SkyFire55

**Comment by se7enek** [comment]
Obviously, in above example players would NOT get rewards for Day 1,2,3,4,5 unless something like:
Day =< 5 is added
or Day = 1 || Day = 2 etc.

**Comment by VimbaTheFish** [comment]
Interesting idea.
I have plans to fully rewrite DailyRewards, coz now it seems a bit...hmm...old. I think i will add this function to new version, coz it seems usable. I will think about structure of this idea.

**Comment by se7enek** [comment]
Not a pro coder, but make it like in if elseif format and store variable in the Day placeholder.

Simple:
Day == 5 { }
Day % 7 == 0 { }
Day > 5 { }

Would be "converted" to:
If (day == 5) {}
Else If (day % 7 == 0) { }
Else If (day > 5) { }

and that's it, i guess.
